htdocs/comm/card.php

```diff
@@ -167,6 +167,7 @@
 		$result = $object->update($object->id, $user, 1, 1, 0);
 		if ($result < 0) {
 			setEventMessages($object->error, $object->errors, 'errors');
+			$_GET['action'] = 'editcustomeraccountancycode';
```

Comment:
setting $_GET looks strange.
Do you mean 
$action  = 'editcustomeraccountancycode' 
?